engine/index.md

```diff
@@ -98,8 +98,8 @@ on the separate [Release Notes page](/release-notes)
 As changes are made to Docker there may be times when existing features
 will need to be removed or replaced with newer features. Before an existing
 feature is removed it will be labeled as "deprecated" within the documentation
-and will remain in Docker for, usually, at least 3 releases. After that time
-it may be removed.
+and will remain in Docker for, usually, at least 3 stable releases (roughly 9 months).
```

Comment:
Can we remove `, usually,`? What are the circumstances in which it would be shorter than 3 months?